cli/tests/upgrade.bats

```diff
@@ -186,6 +186,26 @@ teardown() {
   assert_equal "$curr_lock_hash" "$prev_lock_hash"
 }
 
+@test "upgrade for flake installable" {
+  "$FLOX_BIN" init
+  cat <<EOF | "$FLOX_BIN" edit -f -
```

Comment:
Nitpick: I find all the heredoc stuff hard to reason about. At first glance I have no idea what this is doing with `<<EOF |`.

Reply:
Better?